projects/paraspace/helper/index.js

```diff
@@ -24,7 +24,7 @@ async function tvl(_, _1, _cb, { api, }) {
   if (api.chain === "ethereum") {
     toa.push(...[[Bayc, P2PPairStaking], [Mayc, P2PPairStaking], [Bakc, P2PPairStaking]])
   }
-  return sumTokens2({ chain: api.chain, balances, tokensAndOwners: toa, blacklistedTokens: ['0x0000000000000000000000000000000000000001', address.ethereum.cAPE] })
```

Comment:
I remember adding this because there some some doublecounting at some point. Isnt this a wrapped for staked APE, and that is already included in tvl?